jobsdb/jobsdb.go

```diff
@@ -2001,7 +2000,7 @@ func (jd *HandleT) backupTable(backupDSRange dataSetRangeT, isJobStatusTable boo
 			}
 			contentSlice[idx] = rowBytes
 		}
-		content := bytes.Join(contentSlice[:], []byte("\n"))
+		content := append(bytes.Join(contentSlice[:], []byte("\n")), []byte("\n")...)
```

Comment:
Can we write a comment here?
Why do we need to append \n at the end again.

Reply:
done